modules/logging-loki-pod-placement.adoc

```diff
@@ -0,0 +1,209 @@
+// Module included in the following assemblies:
+// logging/log_storage/cluster-logging-loki.adoc
+
+:_content-type: CONCEPT
+[id="logging-loki-pod-placement_{context}"]
+= Loki pod placement
+You can control which nodes the Loki pods run on and prevent other workloads from using those nodes by using tolerations or node selectors on the pods.
+
+You apply tolerations to the log store pods through the LokiStack custom resource (CR) and apply taints to a node through the node specification. A taint on a node is a `key:value` pair that instructs the node to repel all pods that do not tolerate the taint. Using a specific `key:value` pair that is not on other pods ensures only the log store pods can run on that node.
+
+.Example LokiStack with node selectors
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+spec:
+# ...
+  template:
+    compactor: # <1>
+      nodeSelector:
+        node-role.kubernetes.io/infra: "" # <2>
+    distributor:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+    gateway:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+    indexGateway:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+    ingester:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+    querier:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+    queryFrontend:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+    ruler:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+# ...
+----
+
+<1> Defines the component pod type that the node selector applies to.
+<2> Pods are be moved to nodes containing the label defined.
+
+In the example configuration above all Loki pods are moved to nodes containing the `node-role.kubernetes.io/infra: ""` label.
```

Comment:
[IBMS suggests](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a) to avoid directional prepositions
```suggestion
In the previous example configuration, all Loki pods are moved to nodes containing the `node-role.kubernetes.io/infra: ""` label.
```